src/styl/index.styl

```diff
@@ -278,4 +322,54 @@ section.settings
 
         &.off
             .box
-                background none
\ No newline at end of file
+                background none
+
+section.notes-add, section.notes-edit
+    height 25vh
+    top 40%
+    width 50%
+
+.note-add-input
+    height: 30px;
+    width: 193px;
```

Comment:
I'd vote to replace this hardcoded width with `100%` and add a `flex-grow: 1` to `textfield flex > .note-add-input` to let the input grow.